src/test/java/com/google/devtools/build/lib/syntax/SkylarkEvaluationTest.java

```diff
@@ -1251,6 +1251,19 @@ public void testJavaCallWithPositionalAndKwargs() throws Exception {
         .testLookup("b", "with_params(1, true, false, true, false, a, [\"a\", \"b\"])");
   }
 
+  @Test
+  public void testPositionalArgTypes() throws Exception {
```

Comment:
Can we use Starlark for all new tests of the evaluator? The syntactic cost of Java quotation is far larger than the expression of interest, and every change to the test case forces a recompile.